querydsl-core/src/test/java/com/querydsl/core/testutil/Parallelized.java

```diff
@@ -0,0 +1,40 @@
+package com.querydsl.core.testutil;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.runners.Parameterized;
+import org.junit.runners.model.RunnerScheduler;
+
+public class Parallelized extends Parameterized {
+
+    public Parallelized(Class<?> klass) throws Throwable {
+        super(klass);
+        setScheduler(new ThreadPoolScheduler());
+    }
+
+    private static class ThreadPoolScheduler implements RunnerScheduler {
+        private ExecutorService executor;
+
+        public ThreadPoolScheduler() {
+            int numThreads = Runtime.getRuntime().availableProcessors();
```

Comment:
I tested it with 4 and it works well.
I am not sure if DatabaseMetadata is guaranteed to be threadsafe, but it seems to work here.
Without parallelization it task 1min 23secs for me and parallelized 1min.